Assets/Scripts/GraphService.cs

```diff
@@ -180,8 +181,33 @@ void FixEdgeGroup(IList<Edge> edges, Node nodeFrom, Node nodeTo)
         lineRenderer.positionCount = linePositionsCount;
         lineRenderer.SetPositions(linePositions[i].ToArray());
 
-        var meshCollider = edges[i].gameObject.GetComponent<MeshCollider>();
-        lineRenderer.BakeMesh(meshCollider.sharedMesh, true);
+        UpdateEdgeColliders(edges[i], linePositions[i]);
+      }
+    }
+
+    void UpdateEdgeColliders(Edge edge, IList<Vector3> linePositions)
+    {
+      var edgeColliderTransforms = edge.gameObject.GetComponentsInChildren<Transform>()
+        .Where(t => t.gameObject.name == Constants.ColliderGameObjectName).ToList();
+
+      for (int i = 0; i < edgeColliderTransforms.Count; i++)
+      {
+        if (i < linePositions.Count - 1)
+        {
+          var collider = edgeColliderTransforms[i];
+          var capsule = collider.GetComponent<CapsuleCollider>();
+          capsule.radius = 0.09f;
```

Comment:
looks very similar to EdgeGameObjectFactory.GenerateCollider consider extract as new method